### ISSUE TITLE
Quick Add Folder

### ISSUE THREAD

**Issue by OLLI-S** [bitwarden/desktop#67]
When I edit an entry and select a target folder please add an entry "New Folder" to the drop down list.
When I select this entry, a popup is shown where I can enter the name of the new folder.
This increases the usability a lot (saves many mouse clicks).

**Comment by OLLI-S** [comment]
I can understand that you move to a internal feature list.
But when you move existing topics then I can not filter "all suggestions reported by me".
In GH I can do this.

So is it possible that you change in the internal feature forums the creator of the topic?

**Comment by kspearrin** [comment]
No, but you can easily bookmark topics within the forums for future reference.  

https://community.bitwarden.com/u/__YOURUSERNAME__/activity/bookmarks
